lib/inspec/control_eval_context.rb

```diff
@@ -133,7 +134,16 @@ def to_s
 
         define_method :only_if do |&block|
           return unless block
-          return if @skip_file == true || block.yield == true
+          return if @skip_file == true
```

Comment:
Can't we just set `@skip_file` to false if we are in `@backend.mock_transport?`

Reply:
I don't think we can because Unit tests also use a mock backend. Doing so would make it impossible to test `only_if`.